src/plugins/kibana_usage_collection/server/collectors/core/core_usage_collector.ts

```diff
@@ -115,21 +115,99 @@ export function getCoreUsageCollector(
+      'apiCalls.savedObjectsDelete.namespace.default.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsDelete.namespace.default.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsDelete.namespace.custom.total': { type: 'long' },
+      'apiCalls.savedObjectsDelete.namespace.custom.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsDelete.namespace.custom.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsFind.total': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.default.total': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.default.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.default.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.custom.total': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.custom.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsFind.namespace.custom.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsGet.total': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.default.total': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.default.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.default.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.custom.total': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.custom.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsGet.namespace.custom.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.total': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.default.total': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.default.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.default.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.custom.total': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.custom.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsUpdate.namespace.custom.kibanaRequest.no': { type: 'long' },
+      // Saved Objects Management APIs
       'apiCalls.savedObjectsImport.total': { type: 'long' },
-      'apiCalls.savedObjectsImport.kibanaRequest.yes': { type: 'long' },
-      'apiCalls.savedObjectsImport.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.default.total': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.default.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.default.kibanaRequest.no': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.custom.total': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.custom.kibanaRequest.yes': { type: 'long' },
+      'apiCalls.savedObjectsImport.namespace.custom.kibanaRequest.no': { type: 'long' },
```

Comment:
For each existing SO API, I removed these two usage stats fields in favor of these 6 additional ones. This gives us the ability to tell how many consumers, first- or third-party, are using SO APIs in non-default spaces.